Uno/NuGetPackageExplorer.Shared/InspectPackage.xaml

```diff
@@ -1,1017 +1,877 @@
+                                                                Content="{x:Bind Package.PackageMetadata.Id}"
+                                                                Style="{StaticResource LabeledTextContentControlStyle2}" />
+                                    <wct:HeaderedContentControl Header="{x:Bind pexpl:Resources.Dialog_VersionLabel}"
+                                                                Content="{x:Bind Package.PackageMetadata.Version}"
+                                                                Style="{StaticResource LabeledTextContentControlStyle2}" />
+                                    <HyperlinkButton Content="View Metadata source"
+                                                     Command="{Binding ViewMetadataSourceCommand}"
+                                                     Padding="0" />
+                                </legacyControl:StackPanelEx>
+                                <muxc:Expander Header="Digital Signatures"
+                                               Style="{StaticResource InnerSectionExpanderStyle_v2}"
+                                               IsExpanded="{Binding AreExpandedByDefault}"
+                                               Visibility="{Binding Package.Package.RepositorySignature, Converter={StaticResource NullToCollapsed}}">
+                                    <legacyControl:StackPanelEx Spacing="5">
+                                        <wct:HeaderedContentControl Header="{x:Bind pexpl:Resources.Dialog_PublisherLabel}"
+                                                                    Content="{Binding Package.Package.PublisherSignature}"
+                                                                    ContentTemplate="{StaticResource PublisherSignatureTemplate}"
+                                                                    Visibility="{Binding Package.Package.PublisherSignature, Converter={StaticResource NullToCollapsed}}"
+                                                                    Style="{StaticResource LabeledTextContentControlStyle2}" />
+                                        <wct:HeaderedContentControl Header="{x:Bind pexpl:Resources.Dialog_RepositoryLabel}"
+                                                                    Content="{Binding Package.Package.RepositorySignature}"
+                                                                    ContentTemplate="{StaticResource RepositorySignatureTemplate}"
+                                                                    Visibility="{Binding Package.Package.RepositorySignature, Converter={StaticResource NullToCollapsed}}"
+                                                                    Style="{StaticResource LabeledTextContentControlStyle2}" />
+                                    </legacyControl:StackPanelEx>
+                                </muxc:Expander>
+                                <muxc:Expander Header="Health"
```

Comment:
The expander is now rotating its chevron when closed, I wonder if it's an Uno issue.

Reply:
@jeromelaban I noticed on newer versions of Uno packages the Expander fails to display altogether which is definitely a regression

Reply:
Got it. Could be related to https://github.com/unoplatform/uno/issues/11596 and https://github.com/unoplatform/uno/issues/9062